WooCommerce/Classes/ViewRelated/Orders/Order Details/Address Edit/AddressFormViewModelProtocol.swift

```diff
@@ -257,26 +268,33 @@ open class AddressFormViewModel: ObservableObject {
         self.storageManager = storageManager
         self.stores = stores
         self.analytics = analytics
+        self.featureFlagService = featureFlagService
+        self.noticePresenter = noticePresenter
 
         // Listen only to the first emitted event.
-        onLoadTrigger.first().sink { [weak self] in
+        firstLoadTrigger.first().sink { [weak self] in
             guard let self = self else { return }
             self.bindSyncTrigger()
             self.bindNavigationTrailingItemPublisher()
             self.bindHasPendingChangesPublisher()
 
+            self.trackOnLoad()
+        }.store(in: &subscriptions)
+
+        onLoadTrigger.sink { [weak self] in
+            guard let self = self else { return }
+            self.firstLoadTrigger.send()
+
```

Comment:
This is an interesting change that I would like to know your opinion and if it could be improved 🙂 The idea is to separate the bind logic from the data refresh logic. We only need to create the subscription once to bind the elements, but we need to refresh the data every time the view appears. So I decided to create two triggers, one for the first time and one for the rest. 

The code seems to work, but as I am not a Combine expert, maybe this code could be improved 😅

WDYT? 🤔